src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -621,6 +622,27 @@ describe('Catalogue Items Dialog', () => {
     expect(onClose).not.toHaveBeenCalled();
   }, 10000);
 
+  it('displays recently added section', async () => {
+    vi.useFakeTimers();
+    vi.setSystemTime(new Date("2024-01-09T12:00:00.000+00:00"));
+  
+    props = {
+      ...props,
+      parentInfo: getCatalogueCategoryById('4'),
+    };
+
+    createView();
+    const manufacturerPopup = screen.getAllByRole('combobox')[0];
+    
+    user.type(manufacturerPopup, 'Man')
+    vi.advanceTimersByTimeAsync(2000);
+    const options = await screen.findAllByRole('option');
+    expect(options).toHaveLength(5)
+    expect(screen.getAllByText("Manufacturer B")).toHaveLength(2);
+    expect(screen.getByText("A-Z")).toBeInTheDocument()
+    expect(screen.getByText("Recently Added")).toBeInTheDocument()
```

Comment:
Some of these have semi colons some don't, could be a vscode setting. If you do ctrl+shift+p and type Format with... you can select a default formatter of prettier (you may need the extension). We just have it set to format on save - again if you do ctrl+shift+p and search for settings, then in the settings search for format on save and check it then it should do it automatically.

![image](https://github.com/user-attachments/assets/fce6737e-b2be-41bc-a729-08a836e818eb)
![image](https://github.com/user-attachments/assets/daf3e42a-7972-46d3-a329-70193fd6d31d)

Reply:
Ah, thanks for the advice. I had just been doing it manually, and missing it in a few places. Got it enabled now.